libs/alcotest_ext/tests/Failing_test.ml

```diff
@@ -24,6 +24,8 @@ consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
 cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
 non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 |});
+    t "missing snapshot" ~checked_output:Stdout (fun () ->
+        print_endline "hello");
```

Comment:
nit: why do we print "hello" here?

Reply:
I guess it felt better to print something so as to resemble an ordinary use case, but you're right it's not necessary. I see how it can be distracting. I'll remove it.